2022/pt-br/Days/day05.md

```diff
@@ -43,43 +43,42 @@ Uma vez implantado, vamos operá-lo. E operá-lo pode envolver algo como você c
 
-### Continuous Delivery
+### Entrega Contínua
 
-Continuous Delivery = Plan > Code > Build > Test
+Entrega Contínua = Planejar > Codar > Construir > Testar
 
-### Continuous Integration
+### Integração Contínua
 
-This is effectively the outcome of the Continuous Delivery phases above plus the outcome of the Release phase. This is the case for both failure and success but this is fed back into continuous delivery or moved to Continuous Deployment.
+Este é efetivamente o resultado das fases de Entrega Contínua acima, mais o resultado da fase de Lançamento(Release). Esse é o caso tanto do fracasso quanto do sucesso, mas isso é realimentado na entrega contínua ou movido para a implantação contínua.
 
-Continuous Integration = Plan > Code > Build > Test > Release
+Integração Contínua = Plano > Código > Construir > Teste > Lançamento(Release)
 
-### Continuous Deployment
+### Implantação Contínua
 
-If you have a successful release from your continuous integration then move to Continuous Deployment which brings in the following phases
+Se você tiver uma versão bem-sucedida de sua integração contínua, mude para a implantação contínua, que traz as seguintes fases
 
-CI Release is Success = Continuous Deployment = Deploy > Operate > Monitor
+O lançamento(release) da Integração Contínua foi bem-sucedida = Implantação Contínua = Implantar > Operar > Monitorar
 
-You can see these three Continuous notions above as the simple collection of phases of the DevOps Lifecycle.
+Você pode ver essas três noções Contínuas acima como uma simples coleção de fases do ciclo de vida do DevOps.
 
-This last bit was a bit of a recap for me on Day 3 but think this makes things clearer for me.
+Esta última parte foi uma recapitulação para mim no Dia 3, mas acho que isso torna as coisas mais claras para mim.
 
-### Resources
+### Recursos
 
-- [DevOps for Developers – Software or DevOps Engineer?](https://www.youtube.com/watch?v=a0-uE3rOyeU)
-- [Techworld with Nana -DevOps Roadmap 2022 - How to become a DevOps Engineer? What is DevOps?](https://www.youtube.com/watch?v=9pZ2xmsSDdo&t=125s)
-- [How to become a DevOps Engineer in 2021 - DevOps Roadmap](https://www.youtube.com/watch?v=5pxbp6FyTfk)
+- [DevOps para desenvolvedores – Engenheiro de software ou DevOps?](https://www.youtube.com/watch?v=a0-uE3rOyeU)
+- [Techworld with Nana -DevOps Roadmap 2022 - Como se tornar um engenheiro DevOps? O que é DevOps?](https://www.youtube.com/watch?v=9pZ2xmsSDdo&t=125s)
+- [Como se tornar um engenheiro DevOps em 2021 - DevOps Roadmap](https://www.youtube.com/watch?v=5pxbp6FyTfk)
 
-If you made it this far then you will know if this is where you want to be or not.
+Se você chegou até aqui, saberá se é aqui que deseja estar ou não.
```

Comment:
Obs: In the previous days the translation was "Se você chegou até aqui, saberá se é aqui que você quer estar ou não. Vejo você no Dia 5."
I think "deseja" is more appropriate so let's keep it like that moving forward.